### ISSUE TITLE
Follow `developer_guide.md` & meet errors in precommit.

### ISSUE THREAD

**Issue by Antlera** [intelligent-machine-learning/dlrover#517]
Check my commits by running command as followed:

- ```docker run -v `pwd`:/dlrover -it easydl/dlrover:ci /bin/bash```
- `cd /dlrover`
- `pre-commit run -a`

However, `pre-commit run -a` failed and come up with the log attached.
[precommit_error.log](https://github.com/intelligent-machine-learning/dlrover/files/12136474/precommit_error.log)

**Comment by workingloong** [comment]
You can try to `pip install typed_ast` before `pre-commit run -a`

**Comment by Antlera** [comment]
Fixed by updating mirrors-mypy to v1.2.0 [.pre-commit-config.yaml](https://github.com/intelligent-machine-learning/dlrover/blob/master/.pre-commit-config.yaml)
As followed:
```
  - repo: https://github.com/pre-commit/mirrors-mypy
    rev: v1.2.0
```

**Comment by Antlera** [comment]
> Fixed by updating mirrors-mypy to v1.2.0 [.pre-commit-config.yaml](https://github.com/intelligent-machine-learning/dlrover/blob/master/.pre-commit-config.yaml) As followed:
> 
> ```
>   - repo: https://github.com/pre-commit/mirrors-mypy
>     rev: v1.2.0
> ```

Closed as fixed by modifications in pre-commit.config in my env. Tks for the help from @workingloong.
